vertical-tabbrowser.xml

```diff
@@ -252,15 +265,20 @@
 
               this.tabbrowser.updateWindowResizers();
               break;
-            case "mouseout":
-              // If the "related target" (the node to which the pointer went) is not
+            case 'mouseout':
+              // If the 'related target' (the node to which the pointer went) is not
```

Comment:
I think we could leave the double-quotes in comments…  :wink:
(But don't bother reverting it here.)